src/SFA.DAS.EmployerFinance.Web/Program.cs

```diff
@@ -15,6 +18,13 @@ public static void Main(string[] args)
             WebHost.CreateDefaultBuilder(args)
                 .UseKestrel(options => options.AddServerHeader = false)
                 .UseStartup<Startup>()
-                .UseStructureMap();
+                .UseStructureMap()
+                .ConfigureLogging(logging =>
+                {
+                    logging.ClearProviders();
+                    logging.SetMinimumLevel(LogLevel.Debug);
```

Comment:
Is the minimum level setting needed too or will this be picked up from `nlog.config`?

Reply:
Removed configLogging as in the end they were both not needed. I originally found those setting on the nlog github page